simul.spec

```
@@ -0,0 +1,105 @@
+%global daos_major 0
+
+%global with_mpich 1
+%global with_openmpi3 1
+
+%if %{with_mpich}
+%global mpi_list mpich
+%endif
+%if %{with_openmpi3}
+%global mpi_list %{?mpi_list} openmpi3
+%endif
+
+%if (0%{?suse_version} >= 1500)
+%global module_load() if [ "%{1}" == "openmpi3" ]; then MODULEPATH=/usr/share/modules module load gnu-openmpi; else MODULEPATH=/usr/share/modules module load gnu-%{1}; fi
+%global mpi_libdir %{_libdir}/mpi/gcc
+%else
+%global module_load() module load mpi/%{1}-%{_arch}
+%global mpi_libdir %{_libdir}
+%endif
+
+Name:    simul
+Version: 1.16
+Release: 0%{?commit:.git%{shortcommit}}%{?dist}
```

This should start at 1.

```suggestion
Release: 1%{?commit:.git%{shortcommit}}%{?dist}
```

---

Ok